vector-im/element-android
# [Location share] Update minimum sending period to 5 seconds for a live

## Type of change


## Content


Setting the minimal period of location tracking to 5 seconds. Removing debounce for the first received location to have faster result on UI.

## Motivation and context


Closes #6653 

## Screenshots / GIFs





## Tests



- Start a live in a room
- Emulate a navigation route and run it at the maximal speed
- Check in network logs there is no "Too many requests" errors

## Tested devices

- OS version(s): Android 11

## Checklist
